### PR TITLE
docs: extend contribution doc

### PR DESCRIPTION
**Pull request by patatman**
To make it easier for new comer, and added some tips for OSX plus Visual Studio Code.

Signed-off-by: Patatman <git@jeursen.nl>

# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/2207)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,3 +93,16 @@ Then using the `BUILDKIT_HOST` environment variable before running any `make` ta
 ```bash
 BUILDKIT_HOST=tcp://192.168.1.50:1234 make initramfs
 ```
+
+### Docker for mac
+
+To enable building buildX on Docker for Mac you need to enable the experimental features in the docker app.
+To enable this go to: Docker -> prefrences -> Command Line -> "Enable experimental features" should be toggled on.
```
Comment (rsmitty) [inline CONTRIBUTING.md:100]: "preferences"

**Comment by patatman** [comment]
Maybe I should add https://languagetool.org/ as a Visual Studio code extension 😅

**Pull request merged**